css/style.css

```diff
@@ -344,6 +344,11 @@ body {
     background-color: #efefef;
 }
 
+.floe-news-item img {
+    padding-top: 1.7em;
+    padding-bottom: 0.25em;
+}
+
```

Comment:
@dayotte - could you also add a new style.
```
figcaption details p {
   margin-top: 1rem;
}
```

This will add a bit of margin to the literal description of the images.